composeApp/src/androidMain/kotlin/io/github/couchtracker/tmdb/TmdbLanguage.kt

```diff
@@ -0,0 +1,48 @@
+package io.github.couchtracker.tmdb
+
+/**
+ * A language, compatible with TMDB APIs.
+ * Specification is on https://developer.themoviedb.org/docs/languages
+ *
+ * @property language ISO 639-1
+ * @property country ISO 3166-1
+ */
+data class TmdbLanguage(
+    val language: String,
+    val country: String?,
+) {
+    init {
+        check(language.length == 2) { "Invalid language '$language'" }
```

Comment:
TMDB documentation says:

> The language code system we use is [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). Unfortunately, there are a number of languages that don't have a ISO-639-1 representation. **We may decide to upgrade to ISO-639-3 in the future but do not have any immediate plans to do so.**

I don't seen any plans for their upgrade, but I'm hoping that they're not going to simply start returning ISO-636-3 codes out of the blue on the same field. Were you able to check this?